.github/workflows/CI-staging.yml

```diff
@@ -1,60 +1,192 @@
 # This workflow will install Slips dependencies and run unit tests
-
 name: CI-staging
 
 on:
-    push:
-        branches:
-            # features will be added to this branch using PRs, not need to re-run the tests on push
-            - '!develop'
-            - '!master'
-    pull_request:
-        branches:
-            - 'develop'
-            - '!master'
+  pull_request:
+    branches:
+    - 'master'
```

Comment:
hey @Sekhar-Kumar-Dash why did you change that?
that staging workflow is supposed to be run on PR to develop

the production-testing workflow runs on PR to master
is there a reason why you changed that?

Reply:
Hey @AlyaGomaa  sorry I removed it mistakenly. Thank you for correcting this mistake.